cli/install/template.go

```diff
@@ -177,9 +177,9 @@ spec:
         args:
         - "proxy-api"
         - "-addr=:8086"
-        - "-metrics-addr=:9996"
-        - "-destination-addr=:8089"
-        - "-telemetry-addr=:8087"
+        - "-metrics-addr=127.0.0.1:9996"
```

Comment:
Based on your table in the PR description, I think that metrics should still bind to all interfaces. I'd revert this to:

```
        - "-metrics-addr=:9996"
```

Reply:
Good catch, thanks.  Will revert.